addons/mail/static/src/core/web/thread_model_patch.js

```diff
@@ -103,5 +103,6 @@ patch(Thread.prototype, {
         chatWindow.autofocus++;
         this.state = "open";
         chatWindow.notifyState();
+        this.markAsRead();
```

Comment:
I would expect not needing to manually `markAsRead()` here: opening the chat window while focusing it should automatically put the current user in the condition to mark it as read